app/src/main/java/org/mozilla/fenix/home/recentbookmarks/view/RecentBookmarkItemViewHolder.kt

```diff
@@ -0,0 +1,41 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.home.recentbookmarks.view
+
+import android.view.View
+import kotlinx.android.synthetic.main.recent_bookmark_item.bookmark_title
+import kotlinx.android.synthetic.main.recent_bookmark_item.bookmark_subtitle
+import kotlinx.android.synthetic.main.recent_bookmark_item.bookmark_item
+import kotlinx.android.synthetic.main.recent_bookmark_item.favicon_image
+import mozilla.components.concept.storage.BookmarkNode
+import mozilla.components.support.ktx.kotlin.tryGetHostFromUrl
+import org.mozilla.fenix.R
+import org.mozilla.fenix.ext.components
+import org.mozilla.fenix.ext.loadIntoView
+import org.mozilla.fenix.home.recentbookmarks.interactor.RecentBookmarksInteractor
+import org.mozilla.fenix.utils.view.ViewHolder
+
+class RecentBookmarkItemViewHolder(
+    private val view: View,
+    private val interactor: RecentBookmarksInteractor
+) : ViewHolder(view) {
+
+    fun bind(bookmark: BookmarkNode) {
+        bookmark_title.text = bookmark.title ?: bookmark.url
+        bookmark_subtitle.text = bookmark.url?.tryGetHostFromUrl() ?: bookmark.title ?: ""
```

Comment:
I'm not quite sure what the right subtitle is here. From the examples it looks like it might be a shortened title? 
<img width="693" alt="image" src="https://user-images.githubusercontent.com/43795363/123011499-19641580-d386-11eb-83fa-286bcc769173.png">

Reply:
I think the idea is to grab the `og:image` for the thumbnail, so the subtitle might have been the `og:description` from what I can see in [the metadata](https://ogp.me/#optional) that it can provide.

Probably something for when we can grab those two data elements.

Reply:
Very true. I filed an issue for that - I'm going to give that to @neha-b2001 to work on :)